cypress/e2e/scripts.test.ts

```diff
@@ -10,8 +10,8 @@ beforeEach(() => {
     cy.visit('/')
 })
 
-describe('Analytics', () => {
-    it('has a data layer for GTM and GTM loads', () => {
+describe('Google Tag Manager', () => {
```

Comment:
Not sure why these are showing as changes since this branch & `4.0-launch` are up to date with main 🤔

Reply:
I thought it was git weirdness at first, but then it clicked. 💡 It's because we're merging into the `4.0-launch` branch so it needs to be up to date with that. I updated `4.0-launch` with `main` as well so now everything is up to date.

Reply:
:O Woah -- good find